### ISSUE TITLE
CRUD Service not performing date comparation on nested date fields

### ISSUE THREAD

**Issue by flower-of-the-bridges** [mia-platform/crud-service#237]
**Description**

I have a collection with an array called `ricevute` having the following JSON Schema:

```json
{
  "type": "object",
  "properties": {
     "data": { "type": "string", "format": "date-time" }
  }
}
```

so each record of the collection has a `ricevute` array with the field `dataInizio` having as value a `Date` object.

**Environment**

I'm using latest CRUD Service version 

**Minimal Reproduction**

However, when I perform the following query using the `_q` parameter:

```json
{ "ricevute.data": {"$gte": "<some iso date>" } }
```
the CRUD Service prints the following log:

```json
{"level":20, "hostname":"crud-service","reqId":"req-test","query":{"$and":[{"$and":[{"ricevute.data":{"$gte":"<my date>"}}]},{"__STATE__":"PUBLIC"}]}
```

The date is not converted into a Date object but kept as a string, so the query will fail

**Comment by danibix95** [comment]
This was solved in #252. The fix will be available on the next `v6.x` tag.

**Comment by danibix95** [comment]
Tag [v6.10.0](https://github.com/mia-platform/crud-service/releases/tag/v6.10.0) has been released
